packages/server/src/routes/users.js

```diff
@@ -1,29 +1,39 @@
 const express = require('express');
 
 const router = express.Router({ mergeParams: true });
-const { requireAdminUser, isAuthorized } = require('../lib/access-helpers');
+const { requireAdminUser, isAuthorized, isUserAuthorized } = require('../lib/access-helpers');
 const { sendWelcomeEmail } = require('../lib/email');
 const db = require('../db');
 
 router.post('/', requireAdminUser, async (req, res, next) => {
+    const { user } = req.session;
     if (!req.body.email) {
         res.status(400).send('User email is required');
         return;
     }
 
+    let agencyId = user.agency_id;
+    if (Number.isFinite(req.body.agency)) {
+        agencyId = req.body.agency;
+    }
```

Comment:
Should we just require `agency` in the body and not have this fallback that defaults to creating a user within creator's own agency? Client UI could handle setting that as a default value.

Reply:
Since both cases were covered in the existing unit tests it seemed better to follow the pattern than change it and increase the scope of the PR to more front end changes.